middlewares/retry.go

```diff
@@ -42,6 +42,12 @@ func (retry *Retry) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 		recorder := NewRecorder()
 		recorder.responseWriter = rw
 		retry.next.ServeHTTP(recorder, r)
+
+		// In case of a stream, data has already been sent if code 200
+		if recorder.stream {
```

Comment:
it could be more readable if stream become headerAlreadySent or responseAlreadySent ? WDYT